docs/img/pandas_on_ray_data_ingress.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why it shows like there are multiple variants of IO classes for `PandasOnRayFactory` when actually there's [only one variant - `PandasOnRayIO`](https://github.com/modin-project/modin/blob/72a3e51aeeae3465a1a3076d962aef95af689dde/modin/core/execution/dispatching/factories/factories.py#L427-L435)?
![image](https://user-images.githubusercontent.com/62142979/144134793-8a58d6f0-e5a4-47df-93c0-2d00c0a4137e.png)

Reply:
We want to stress that there are multiple implementations of IO classes which are responsible for parallel read/write. At the same time we use the only arrow between the factory and the IO class which indicates the factory exposes only one concrete IO class.